eslint-custom-rules/no-raw-text.js

```diff
@@ -0,0 +1,35 @@
+const ALLOWED_JSX_TAGS = [
+  /^.*Text$/,
+  /^Styled+(Hero|Title[1-4]|Body|ButtonText(|NeutralInfo|Primary|Secondary)|Caption(|NeutralInfo|Primary|Secondary))$/,
+]
+
+const WHITESPACES_REGEX = /^\s+$/
+
+const checkJSXTextAreEmpty = (child) =>
+  child.type !== 'JSXText' || (child.type === 'JSXText' && WHITESPACES_REGEX.test(child.value))
+
+module.exports = {
+  name: 'no-raw-text',
+  meta: {
+    docs: {
+      description:
+        'We use a custom no-raw-text instead react-native/no-raw-text because we have some specific text tags',
+    },
+  },
+  create(context) {
+    return {
+      JSXElement: (node) => {
+        const openingElement = node.openingElement.name
+
+        if (openingElement.object && openingElement.object.name === 'Typo') return
```

Comment:
je pense que c'est simplifiable

```suggestion
        if (openingElement?.object?.name === 'Typo') return
```